src/main/java/at/hannibal2/skyhanni/config/features/Garden.java

```diff
@@ -275,6 +275,12 @@ public class Garden {
     @ConfigAccordionId(id = 7)
     public int cropMilestoneShowOnlyBest = 10;
 
+    @Expose
+    @ConfigOption(name = "Expand top list by current Crop", desc = "Add active crop to the end of the list by expanding the set limit")
```

Comment:
A title that long is not really readable with the current render logic. Please consider using a shorter title.
![grafik](https://user-images.githubusercontent.com/24389977/234540984-f97e4c8b-6246-4992-9c40-8c0f80affd07.png)

Reply:
How about **"If enabled the current farmed crop will be displayed at the end of the list regardless of it's ranking and the set top limit, else it is only visible if its ranking matches the limit"** will this make it clear? What do you think should the character limit be for the title? **"Expand top list"** could fit, too.

Reply:
"Always show current crop" would work even better ig